ndk/sample/src/main/cpp/ndk-sample.cpp

```diff
@@ -22,4 +22,29 @@ JNIEXPORT void JNICALL Java_io_sentry_ndk_sample_NdkSample_message(JNIEnv *env,
     sentry_capture_event(event);
 }
 
+JNIEXPORT void JNICALL Java_io_sentry_ndk_sample_NdkSample_transaction(JNIEnv *env, jclass cls) {
```

Comment:
Since the NDK has no transport, we can't really test this just by looking at Sentry dashboards. A way to check if the envelope is created properly, is by looking at `data/data/io.sentry.ndk.sample/files/outbox`. 

This is probably how we want to this test on the native repo (and then have a integration test similar to [this one from native](https://github.com/getsentry/sentry-native/blob/master/tests/test_integration_http.py#L525) on the java-SDK side)

Reply:
Yeah, this is fine. The `outbox` is the primary interface back to `sentry-android` and, as such, the ideal test boundary. Adding additional machinery beyond parsing the output won't provide better feedback.